Automattic/jetpack
# Admin Page: Rename modulesettingsform High Order component to withModuleSettingsFormHelpers for clarity

This is just a rename for clarity

#### Changes proposed in this Pull Request:


* Updates the name of the High Order Component to be more clear
* Renames the file defining it for consistency.


#### Testing instructions:

* Checkout this branch
* Run `yarn clean-client`.
* Run `yarn build`.
* Confirm it builds properly.
* Load the admin page, settings tab.
* Confirm everything still works

#### Proposed changelog entry for your changes:


* None needed
